packages/utilities/src/utilities/settings/settings.js

```diff
@@ -8,6 +8,7 @@
  *
  */
 const settings = {
+  version: 'dds.v0.5.0',
```

Comment:
Just a thought, but could we grab the versions of the packages themselves?

```javascript
const settings = {
   versions: {},
   stablePrefix: 'dds'
}

setPackageVersion('@carbon/ibmdotcom-react');
setPackageVersion('@carbon/ibmdotcom-styles');
setPackageVersion('@carbon/ibmdotcom-services');
setPackageVersion('@carbon/ibmdotcom-utilities');

function setPackageVersion (packageName) {
  try () {
    settings.versions[packageName] = require(`#{packageName}/package.json`).version;
  catch { /* fail silently */ }
}
```

Reply:
Since we're on independent versioning, this probably won't work after awhile. We will need to be able to grab the global package version somehow.